example/lib/proxy.dart

```diff
@@ -8,18 +8,19 @@ void main() async {
   dio.options
     ..headers['user-agent'] = 'xxx'
     ..contentType = 'text';
-  // dio.options.connectTimeout = 2000;
-  // More about HttpClient proxy topic please refer to Dart SDK doc.
-  dio.httpClientAdapter = IOHttpClientAdapter()
-    ..onHttpClientCreate = (HttpClient client) {
+  dio.httpClientAdapter = IOHttpClientAdapter(
+    onHttpClientCreate: (HttpClient client) {
       client.findProxy = (uri) {
-        //proxy all request to localhost:8888
+        // Proxy all request to localhost:8888.
+        // Be aware, the proxy should went through you running device,
+        // not the host platform.
```

Comment:
I added this because there are multiple times that some users ask: "Why I cannot access my proxy server at 127.0.0.1?". This might help them quickly realize they are on different devices.